unstructured/chunking/title.py

```diff
@@ -287,6 +310,7 @@ def wrapper(*args: _P.args, **kwargs: _P.kwargs) -> List[Element]:
                     combine_text_under_n_chars=params.get("combine_text_under_n_chars", 500),
                     new_after_n_chars=params.get("new_after_n_chars", 500),
                     max_characters=params.get("max_characters", 500),
+                    overlap=params.get("overlap", 0),
```

Comment:
Just trying to think about the way this behaves with the other inputs:
1. what happens if overalp == max characters?
2. overlap > max characters - should error before going into chunking

Any other things we should be handling?

Reply:
Yes, that's a good point. I can add a validation step for that.